src/containers/ChartInfo/index.js

```diff
@@ -188,4 +203,5 @@ ChartInfo.defaultProps = {
   fetchMonthWeekPpms: () => {},
   fetchYearPpms: () => {},
   fetchAllPpms: () => {},
+  match: null,
```

Comment:
Could this be an object ? like the following
````javascript
{
params: null
}
````
That way we guarantee that the match object will have always have a `params` prop we we don't have to check against.